packages/wrangler/src/cli.ts

```diff
@@ -1,5 +1,35 @@
+import * as Sentry from "@sentry/node";
+import process from "process";
 import { hideBin } from "yargs/helpers";
+import { RewriteFrames } from "@sentry/integrations";
+
 import { main } from ".";
+import * as pkj from "../package.json";
+
+Sentry.init({
+  release: `${pkj.name}@0.0.11`,
+  initialScope: {
+    tags: { [pkj.name]: pkj.version },
+  },
+  dsn: "https://87383452498e420ebdb01de6100e44ec@o226679.ingest.sentry.io/6139281",
+  tracesSampleRate: 1.0,
+  integrations: [
+    new RewriteFrames({
+      root: "",
+      prefix: "/",
+    }),
+    new Sentry.Integrations.Http({ tracing: true }),
+  ],
+});
+
+process.on("uncaughtExceptionMonitor", (err, origin) => {
```

Comment:
Do we ever get an unhandled exception? Below we have a `.catch()` handler on the call to `main()`.